.gitignore

```diff
@@ -9,6 +9,9 @@ venv
 .vscode
 .coverage
 /.tox
+.env
+.envrc
+.secrets
```

Comment:
I have no objection to adding these to .gitignore, and generally no objection to small drive-bys. However, I think for .gitignore it's cleaner if changes are in dedicated (or related) PRs.

Reply:
Either works for me. If you decide to keep them here, can you please add a PR comment as to why these are needed?

Reply:
Done